source/index.md

```diff
@@ -600,9 +602,50 @@ object][ApplePay-PaymentToken] for more information.
   subsequent recurring authorization.
 </p>
 
-##### Method: `mobilepayonline`
 
-<dl class="dl-horizontal">
+---
+##### **Method**: `googlepay`
+
+To accept a payment using Google Pay, apart from the complete payment method
+token and merchant ID, the derived shared secret is required.
+Please refer to the [official documentation][GooglePay-PaymentCryptography].
+Supported protocol versions are `ECv1` and `ECv2`.
+
+<dl>
+  <dt>googlepay[token]</dt>
+  <dd>[:json:]<br />Raw payment method token as received in response from Google. UTF-8
+    encoded serialization of a JSON dictionary.</dd>
+  <dt>googlepay[shared_secret]</dt>
+  <dd>[:base64:]<br />The shared secret derived from the ephemeral public key and
+    your private key.</dd>
+  <dt>googlepay[merchant_id]</dt>
+  <dd>[\x21-\x7E]+
+    <i><a target="_blank" href="http://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters">ASCII printable characters exluding space</a></i>
+    <br />The Google Pay ID of the merchant.</dd>
+</dl>
+
+<p class="alert alert-info">
+  <b>Notice:</b> Signing is required to use the <code>googlepay</code> payment
+  method.
+  <br />
+  <b>Notice:</b> An authorization made with <code>googlepay</code> may be fully
+  3-D Secured, 3-D Secure attempted, or with no 3-D Secure; this is indicated by
+  the <code>eciIndicator</code> of the <code>paymentMethodDetails</code>.
+  <br />
+  <b>Notice:</b> An authorization made with <code>googlepay</code> is
+  strongly authenticated (SCA in PSD2) if fully 3-D Secure.
+  <br />
+  <b>Notice:</b> An authorization made with <code>googlepay</code> cannot be a
+  subsequent recurring authorization.
```

Comment:
```suggestion
  subsequent recurring authorization. If marked as <code>recurring</code>, the
  authorization will be an initial recurring authorization.
```

Reply:
This has been fixed by mentioning it earlier.